client/gulpfile.js

```diff
@@ -85,13 +85,26 @@ const javascriptWatch = () =>
     .pipe(gulp.dest("dist/app"))
     .pipe(connect.reload());
 
-const worker = () =>
+const prepWorker = (cb) => {
+    // cp wasm_exec.js to be glued
+    execSync(
+      "cp \"$(go env GOROOT)/misc/wasm/wasm_exec.js\" src/worker"
+    );
+    // export as Go module
```

Comment:
LGTM, however I'm not sure we need to modify the file with an export anymore. Maybe we can change from `import Go from "wasm_exec";` to `import "wasm_exec";` since loading the file already sets Go as a global variable. Though we would need some changes to the test suite since some tests import go as well. But this PR is good already. We could update the import and tests in a separate PR :+1:

Reply:
I have updated and all tests pass, so merging.